common/src/main/java/de/maxhenkel/voicechat/config/ClientConfig.java

```diff
@@ -60,151 +60,151 @@ public ClientConfig(ConfigBuilder builder) {
-                        "If the mod should check for microphone permissions (MacOS only)"
+                        "If the mod should check for microphone permissions (macOS only)"
                 );
         showFakePlayersDisconnected = builder
                 .booleanEntry("show_fake_players_disconnected", false,
                         "If fake players should have the disconnected icon above their head"
                 );
         offlinePlayerVolumeAdjustment = builder
                 .booleanEntry("offline_player_volume_adjustment", false,
-                        "If the volume adjustment GUI should also show offline players"
+                        "If the volume adjustment interface should also display offline players"
                 );
         audioType = builder
                 .enumEntry("audio_type", AudioType.NORMAL,
                         "The 3D audio type",
-                        "Possible values are 'NORMAL', 'REDUCED' and 'OFF'"
+                        "Valid values are 'NORMAL', 'REDUCED', and 'OFF'"
                 );
         useNatives = builder
                 .booleanEntry("use_natives", true,
                         "If the mod should load native libraries",
-                        "If set to false, the Java Opus implementation will be used, the denoiser won't be available and you won't be able to record audio."
+                        "When disabled, the Java Opus implementation will be used instead, the denoiser won't be available, and you won't be able to record the voice chat audio"
                 );
         freecamMode = builder
                 .enumEntry("freecam_mode", FreecamMode.CAMERA,
-                        "How listening to other players should work when using freecam mods",
-                        "Possible values are 'CAMERA' and 'PLAYER'",
-                        "CAMERA: You will hear voice chat audio around your camera. Whether you hear distant audio depends on the voice chat broadcast range of the server",
-                        "PLAYER: You will hear voice chat audio around your player no matter where your camera is"
+                        "Specifies how listening to other players should work when using freecam mods",
+                        "Valid values are 'CAMERA' and 'PLAYER'",
+                        "CAMERA: You will hear the voice chat around your camera. Whether you will still be able to hear the voice chat when the camera is far away from your character depends on the voice chat broadcast range of the server",
+                        "PLAYER: You will hear the voice chat around your character no matter where your camera is"
                 );
         muteOnJoin = builder
                 .booleanEntry("mute_on_join", false,
-                        "If enabled, you will be automatically muted when joining a world"
+                        "Once enabled, you will be automatically muted when joining a world"
```

Comment:
I prefer "if" here

Reply:
Resolved locally. I will publish everything at once after we've discussed all the new issues you've brought up.